test_data/makedata_suites/005_check_system_collection.js

```diff
@@ -2,8 +2,13 @@
 
 (function () {
   return {
-    isSupported: function (version, oldVersion, options, enterprise, cluster) {
-      return true;
+    isSupported: function (currentVersion, oldVersion, options, enterprise, cluster) {
+      if(currentVersion.includes("nightly")){
+        let version = semver.parse(currentVersion.split('-')[0]);
+        return semver.satisfies(currentVersion, '>3.11.2 <3.12.0 || >=3.12.1');
```

Comment:
```
 >=3.12.0'
```
?

Reply:
The idea is to let tests pass on 3.12.0-nightly because this is a known issue, but we expect it to be fixed by 3.12.0 release. So tests will only run on 3.12.0 or 3.12.1-nightly.

Reply:
ah, got it. nice trick ;)